src/app/banking/vouchers/search-redeemed-vouchers.component.ts

```diff
@@ -33,13 +33,15 @@ export class SearchRedeemedVouchersComponent
   }
 
   protected getFormControlNames(): string[] {
-    return ['types', 'periodBegin', 'periodEnd'];
+    return ['types', 'periodBegin', 'periodEnd', 'operator'];
   }
 
   protected toSearchParams(value: any): UserVoucherSearchParams {
     const params = cloneDeep(value);
     params['user'] = this.ApiHelper.SELF;
     params['relation'] = VoucherRelationEnum.REDEEMED;
+    delete params['operator'];
```

Comment:
Also, you don't need to delete any mismatching variable, because the generated services will only send known parameters.